### PR TITLE
Update Laser and 3D to be two quizzes

### PR DESCRIPTION
**Pull request by git-wil**

### CODE REVIEW

**Comment by git-wil** [comment]
This should be all the changes necessary to update the quizzes, let me know if you think I missed anything. I will note that in testing, the checkouts page shows an "undefined" quiz for anyone who passed the old Laser3D quiz. I'm fairly sure this is just a remnant of the passed quizzes of each user, but that should reset on August 1 and (I believe, ideally) shouldn't be a problem.

File: MAKE-website/kiosks/scripts/management.js [hunk 1]
```diff
@@ -2601,43 +2601,6 @@ function generateProficiencyDivs(users) {
     return divs;
 }
 
-function generateStatsDivs(users) {
```
Comment (IonImpulse) [inline MAKE-website/kiosks/scripts/management.js:2604]: omg the old stats page lol

File: MAKE-website/scripts/page_quiz_info.js [hunk 2]
```diff
@@ -67,11 +107,21 @@ const quizzes = [
 
 function renderQuizInfo() {
     const quiz_container = document.getElementById("quiz-list-container");
+    const general_quiz_container = document.getElementById("general-quiz-container");
 
     let divs = [];
 
     for (let quiz of quizzes) {
-        divs.push(generateQuizDiv(quiz));
+        let quiz_div = generateQuizDiv(quiz);
```
Comment (IonImpulse) [inline MAKE-website/scripts/page_quiz_info.js:115]: def change this to a find function for the general quiz, then append the rest checking to see if it's general and skipping if it is

**Review by IonImpulse — CHANGES_REQUESTED** [review]
change that one thing for the quizzes and you'll be good to go

File: MAKE-website/scripts/page_quiz_info.js (hunk 2 above)
Comment (git-wil) [reply MAKE-website/scripts/page_quiz_info.js]: Does this commit 8f3b29b011041601feae96e5af7b17148ac06527 have the changes you had in mind?

Comment (IonImpulse) [reply MAKE-website/scripts/page_quiz_info.js]: yup!

**Review by IonImpulse — APPROVED** [review]

**Pull request merged**